lib/dynflow/web/console.rb

```diff
@@ -37,6 +37,15 @@ class Console < Sinatra::Base
         erb :worlds
       end
 
+      post('/worlds/pending_steps') do
+        @worlds = world.coordinator.find_worlds
+        @worlds.each do |w|
+          steps = world.get_pending_steps(w.data['id'], nil, 5).value!.values.sum
```

Comment:
I'm getting ```NoMethodError at /worlds/pending_steps undefined method `sum' for []:Array``` when running pure dynflow, without Rails, as this methods comes from ActiveSupport

Reply:
Actually it was added to pure ruby in 2.4.0, will fix anyway.